src/data/uma-nickname.csv

```diff
@@ -142,13 +142,13 @@
 "真・女王の精神","True・Queen's Spirit"
 "時代の寵児","Era's Favorite"
 "勝利の探究者","Victory Seeker"
-"切磋琢磨",""
-"チームの繋がり",""
-"伝説の証人",""
-"チームの絆",""
+"切磋琢磨","Diligent Application"
```

Comment:
This can have several different meanings - there's a lack of context to decide whether the one used here is correct.

Reply:
Judging by the requirement for it (3 special trainings at once) I would rename it as Friendly Rivalry or Mutual Improvement then.

Reply:
That sounds good.